pkg/tempopb/tempo.pb.go

```diff
@@ -28,7 +28,10 @@ var _ = math.Inf
 const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 
 type TraceByIDRequest struct {
-	TraceID []byte `protobuf:"bytes,1,opt,name=traceID,proto3" json:"traceID,omitempty"`
+	TraceID        []byte `protobuf:"bytes,1,opt,name=traceID,proto3" json:"traceID,omitempty"`
+	BlockStart     []byte `protobuf:"bytes,2,opt,name=blockStart,proto3" json:"blockStart,omitempty"`
```

Comment:
block start/end are guids.  i think we should just use strings b/c it would make things easier to debug and easier to manually query.  this impacts my comment above about choosing default params.  i suggested 0x00/0xFF but it would be a guid string instead that we'd default to.